src/Stan/Report/Css.hs

```diff
@@ -76,6 +77,10 @@ stanCss = do
     ".severityWarning" ? backgroundColor yellow
     ".severityError" ? backgroundColor red
 
+    blockquote ? do
+        paddingLeft (2%)
+        borderLeft solid (px 4) darkGrey
```

Comment:
Let's use `box-shadow` to have double borderline with grey and light grey at once :nail_care: